jit/v2/JIT_v2.0.0.yaml

```diff
@@ -2086,14 +2086,47 @@ components:
     #######
     # Moves
     #######
+    MovesList:
+      type: array
+      description: |
+        An array of `Moves`. Can be per VSA-Partner or as an accumulated result.
```

Comment:
An array of `Moves`. Each object in the array can represent moves data for a specific VSA partner in the vessel or as an accumulated result for the remaining VSA partners in the vessel.